src/Elements/Chips/APIChip.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import classes from "./Chips.module.css";
+
+const APIChip = () => {
+  return (
+    <div>
+      <div className={classes.api}>API</div>
+    </div>
+  );
+};
+
+export default APIChip;
```

Comment:
I think perhaps instead of having each Chip as an individual component, you could create one SkillChip component and pass in the desired colour and name of the skill. This would save a lot of repetition.

Reply:
Similar to what you did with your Tech Card component.